admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -198,11 +204,17 @@ object DatabaseTablesDAO {
         column <- columns
         columnName = column.name
         cellData = {
-          if (settings.photoColumn.contains(columnName)) {
-            // TODO: we're doing this operation two times (get id string), find another way
-            val rowId = resultSet.getString(settings.primaryKeyField)
-            s"$baseUrl/admin/images/${settings.tableName}/$rowId"
-          } else resultSet.getString(columnName)
+          val maybe = settings.columnTypeOverrides.find(_._1 == columnName)
```

Comment:
Ues `.get` instead

Reply:
but it will fail if the type overrides is empty or doesn't have a setting for this column. In this part we're looking if the column needs to be override, there is a case when we don't want that (on the majority of columns)

Reply:
`get` returns `Option`:

```
scala> val map = Map("s" -> 1)
val map: Map[String, Int] = Map(s -> 1)
                                                                                                                                                                     
scala> map.get("s")
val res0: Option[Int] = Some(1)
```